pg_yregress/test.yml

```diff
@@ -26,6 +26,9 @@ instances:
     hba: |
       local all all trust
       host all all all trust
+  encoding_locale:
+    encoding: EUC_KR
+    locale: ko_KR
```

Comment:
need some encoding and locale which will be present in CI runners as well

Reply:
@suryaoruganti I wonder if you have thoughts on locales availability? Or is this how GH worker images are, and they replicated as closely as possible?

Reply:
Our runners are built from the same source as GitHub upstream. However, there could be some additional configs that are not a part of the open source that GitHub could be doing. I'll check this.

Thanks for bringing this to my notice.